sdk/eventhub/Microsoft.Azure.EventHubs/tests/Infrastructure/TestConstants.cs

```diff
@@ -9,11 +9,21 @@ internal static class TestConstants
     {
         // Environment Variables
         internal const string EventHubsConnectionStringEnvironmentVariableName = "EVENT_HUBS_CONNECTION_STRING";
-        internal const string StorageConnectionStringEnvironmentVariableName = "EVENT_HUBS_STORAGE_CONNECTION_STRING";
+        internal const string StorageConnectionStringEnvironmentVariableName = "EVENT_HUBS_STORAGE_CONNECTION_STRING";      
+        
+        //Following 6 const copied from Track two TestEnvironment.cs
+        internal const string EventHubsSubscriptionEnvironmentVariableName = "EVENT_HUBS_SUBSCRIPTION";
+        internal const string EventHubsResourceGroupEnvironmentVariableName = "EVENT_HUBS_RESOURCEGROUP";
+        internal const string EventHubsNamespaceEnvironmentVariableName = "EVENT_HUBS_NAMESPACE";
+        internal const string EventHubsTenantEnvironmentVariableName = "EVENT_HUBS_TENANT";
+        internal const string EventHubsClientEnvironmentVariableName = "EVENT_HUBS_CLIENT";
+        internal const string EventHubsSecretEnvironmentVariableName = "EVENT_HUBS_SECRET";
 
+
+        //Unused after Track two test infrastructure ported
```

Comment:
Agree this would be unused; please remove instead of commenting out.   They can always be restored from source control history if need be.